app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.kt

```diff
@@ -313,54 +290,42 @@ class CreateEventActivity : ActivityForAccessingTumOnline<CreateEventResponse>(R
         }
         binding.eventEndDateView.setOnClickListener {
             hideKeyboard()
-            ThemedDatePickerDialog(this, { _, year, month, dayOfMonth ->
+            DatePickerDialog(this, { _, year, month, dayOfMonth ->
```

Comment:
Does this look good in the dark+light mode? (afaik that was the reason why ThemedTimePickerDialog was introduced)